src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -565,13 +564,20 @@ protected void getChanges2(ChangeEventSourceContext context,
                             if (m.getOp() == Op.SAFEPOINT) {
                                 continue;
                             }
+                            
+                            String pgSchemaNameInRecord = m.getPgschemaName();; 
+                            TableId tempTid;
+                            if(connectorConfig.isYSQLDbType()) {
+                                // This is a hack to skip tables in case of colocated tables
+                                tempTid = YugabyteDBSchema.parseWithSchema(message.getTable(), pgSchemaNameInRecord);
 
-                            String pgSchemaNameInRecord = m.getPgschemaName();
-
-                            // This is a hack to skip tables in case of colocated tables
-                            TableId tempTid = YugabyteDBSchema.parseWithSchema(message.getTable(), pgSchemaNameInRecord);
+                            } else {
+                                tempTid = YugabyteDBSchema.parseWithKeyspace(message.getTable(), //TODO: Check if this breaks the above hack 
```

Comment:
I think we might not require the else block since colocation is only for YSQL tables.